Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -1326,6 +1336,7 @@
 		D31EF9741E9517C600450162 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ARCHS = "$(ARCHS_STANDARD)";
```

Comment:
As discussed offline, we will keep this one and add this line to the other two watch targets.